config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -254,7 +254,7 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api
       testgrid-tab-name: capi-pr-e2e-full-main
-  - name: pull-cluster-api-e2e-workload-upgrade-1-25-latest-main
+  - name: pull-cluster-api-e2e-workload-upgrade-1-26-latest-main
```

Comment:
KUBERNETES_VERSION_UPGRADE_FROM and KUBERNETES_VERSION_UPGRADE_TO must be changed, eventually also ETCD_VERSION_UPGRADE_TO and COREDNS_VERSION_UPGRADE_TO
(same for all the branches, not for main and 1.2 only)